data/json/npcs/talk_tags.json

```diff
@@ -68,6 +68,75 @@
     "category": "<name_b>",
     "//": "generic negative/hostile pronouns",
     "text": [
+      "airhead",
+      "arse",
+      "ass",
+      "assclown",
+      "assface",
+      "asstown",
+      "asswagon",
+      "bitch",
+      "bootlicker",
+      "bugger",
+      "butchery refuse",
+      "butthead",
+      "butt-licker",
+      "chucklefuck",
+      "coward",
+      "cuck",
+      "cunt",
+      "damn <name_b>",
+      "damned <name_b>",
+      "dildo",
+      "dimwit",
+      "dog",
+      "dumbshit",
+      "dumpster",
+      "dumpster fire",
+      "dunderfuck",
+      "effing <name_b>",
+      "emergency ration",
+      "empty headed <name_b>",
+      "expendable <name_b>",
+      "fart",
+      "fat-ass",
+      "frail-ass <name_b>",
+      "frigger",
+      "fungus eater",
+      "goddamn <name_b>",
+      "good for nothing",
+      "inbred",
+      "lazy",
+      "lazy ass",
+      "limp-dick",
+      "low-life",
+      "meat face",
+      "mi-go hugger",
+      "mushy pizza",
+      "pissbrain",
+      "pool noodle",
+      "prick",
+      "pussy",
+      "shitweasel",
+      "slime hugger",
+      "smooth brain",
+      "snowflake",
+      "toilet licker",
+      "tool",
+      "trash",
+      "turd",
+      "twat",
+      "twerp",
+      "useless",
+      "wanker",
+      "waste",
+      "waste of ammo",
+      "weapon holder",
+      "weiner",
+      "weird <name_b>",
+      "wet sandwich",
+      "wet sock",
+      "z fodder",
```

Comment:
Same here.

Reply:
sorry for getting back to that only just now but couldn't those conversion mods just also remove those snippets?

Reply:
Perhaps a new `<foe>` to insert as appropriate?

Reply:
could definitely do that if that is wanted